NDTensors/test/ITensors/TestITensorDMRG/TestITensorDMRG.jl

```diff
@@ -4,14 +4,15 @@
 module TestITensorDMRG
 using ITensors
 using NDTensors
+using NDTensors.CUDAExtensions: cu
```

Comment:
@mtfishman I have now in `NDTensors/imports.jl`, `using .CUDAExtensions: cu` so I could have this (and other statements) written as `NDTensors: cu` Do you have a preference on which one to prefer?

Reply:
I think it is better the way you currently wrote it. In general, it is best to load a function into a namespace from where it is actually defined, as you did here.